test/functional/api/extmark_spec.lua

```diff
@@ -110,6 +110,22 @@ describe('API/extmarks', function()
        pcall_err(set_extmark, ns, marks[2], 0, 0, { end_col = 1, end_row = 1 }))
   end)
 
+  it("can end extranges past final newline when strict mode is false", function()
+    set_extmark(ns, marks[1], 0, 0, {
+      end_col = 1,
+      end_row = 1,
+      strict = false
+    })
+  end)
+
+  it("can end extranges past final column when strict mode is false", function()
+    set_extmark(ns, marks[1], 0, 0, {
+      end_col = 1,
+      end_row = 6,
```

Comment:
this doesn't test what is described, maybe `end_row=0, end_col=6` ?

Reply:
done